frontend/src/app/testResults/operations.graphql

```diff
@@ -98,39 +98,49 @@ query GetFacilityResultsForCsvWithCount(
   }
 }
 
-query GetFacilityResultsMultiplexWithCount(
+query GetResultsMultiplexWithCount(
   $facilityId: ID
   $patientId: ID
   $result: String
   $role: String
+  $disease: String
   $startDate: DateTime
   $endDate: DateTime
   $pageNumber: Int
   $pageSize: Int
 ) {
-  testResultsPage(
+  resultsPage(
     facilityId: $facilityId
     patientId: $patientId
     result: $result
     role: $role
+    disease: $disease
     startDate: $startDate
     endDate: $endDate
     pageNumber: $pageNumber
     pageSize: $pageSize
   ) {
     content {
-      internalId
+      id
+      dateAdded
       dateTested
-      results {
-        disease {
-          name
-        }
-        testResult
-      }
+      disease
+      testResult
       correctionStatus
+      reasonForCorrection
+      facility {
+        name
+        isDeleted
+      }
       deviceType {
         internalId
         name
+#        manufacturer
+#        model
+#        swabTypes {
+#          internalId
+#          name
+#        }
```

Comment:
remove commented out code

Reply:
👍 removed in https://github.com/CDCgov/prime-simplereport/pull/6682/commits/34346c5a58c35cca7a4fde47695fcf9e994ce8ae.